docs/keys_and_certificates.md

```diff
@@ -0,0 +1,293 @@
+# Keys and certificates
+
+Many different keys and certificates are used by all of Fobnail's components.
+Most of them are automatically generated, but some must be provided by user.
+
+## List of keys and certificates used
+
+Refer to [architecture doc](architecture.md) for description of actors and
+phases listed below. List items have the following format:
+
+- **Name of key or certificate**
+    - **Phase during which it is obtained, received or used**
+    - **Other details, if any**
+
+Note that this list includes possible usages that may not be implemented yet.
+
+### Platform Owner
+
+#### Knows in advance
+
+- Platform Owner key (signing and encryption)
```

Comment:
We should be free to use any of those. Instructions in OpenSSL section use RSA, so it can encrypt. Also, I think this falls into `this list includes possible usages`. For concrete examples that depend on use case I think we need a separate document. This one is already pretty long.